android/jni/com/mapswithme/platform/SocketImpl.cpp

```diff
@@ -0,0 +1,35 @@
+#include "../core/jni_helper.hpp"
+#include "base/logging.hpp"
+#include "patform/socket.hpp"
+
+class SocketImpl
+{
+  SocketImpl()
+  {
+    // create m_self
+  }
+
+  ~SocketImpl()
+  {
+    // close socket
+    // delete m_self
+  }
+
+  bool Open(string const & host, uint16_t port) { return true; }
+  bool Close() { return true; }
+  bool Read(uint8_t * data, uint32_t count) { return true; }
+  bool Write(uint8_t const * data, uint32_t count) { return true; }
+private:
```

Comment:
clang-format is wrong here - it should honor single blank lines. Of course, we need to configure it.